server/container/guice/cassandra-guice/src/main/java/org/apache/james/CassandraJamesServerMain.java

```diff
@@ -131,6 +132,7 @@
         new CassandraSessionModule(),
         new CassandraSieveRepositoryModule(),
         new CassandraUsersRepositoryModule(),
+        new PeriodicalHealthChecksModule(),
```

Comment:
I would be in favor of installing PeriodicalHealthChecksModule  in CommonServicesModule

Reply:
I'm still in favor of that.

Do you need some help?

Reply:
Why is this comment hidden? I only saw it just now when browsing the commits tab.